developer/src/common/include/kmn_compiler_errors.h

```diff
@@ -41,19 +41,20 @@
 
 // Message codes
 //
-// All errors here and below are mirrored in kmc-kmn/src/compiler/messages.ts;
-// if you add a new error here be sure to update that file also. Note that this
-// correlation is currently maintained manually. All values must be below
-// 0x1000 (exclusive of severity code).
+// All errors here and below are mirrored in
+// kmc-kmn/src/compiler/kmn-compiler-messages.ts; if you add a new error here be
+// sure to update that file also. Note that this correlation is currently
+// maintained manually. All values must be below 0x1000 (exclusive of severity
+// code).
 #define CERR_None                                          0x00000000
 #define CERR_EndOfFile                                     0x00000001
 
 #define CERR_BadCallParams                                 0x00008002
 #define CERR_CannotAllocateMemory                          0x00008004
-#define CERR_InfileNotExist                                0x00008005
-#define CERR_CannotCreateOutfile                           0x00008006
+#define CERR_InfileNotExist                                0x00004005   // #10678: reduced from fatal to error in 17.0
+// #define CERR_CannotCreateOutfile                           0x00004006   // #10678: reduced from fatal to error in 17.0, but unused
```

Comment:
Wait, so this... can't happen anymore?  Why is it now unused?  

Surely a destination file could, in theory, be in a read-only open mode; how do we now avoid that being a problem?

Reply:
File writing is no longer the responsibility of this component.